src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -163,12 +166,12 @@ def _create_response(self, issue=None, data=None):
             errors_list.append(response)
         return Response({"errors": errors_list})
 
-    def _get_filename_and_path(self, exception, frame_idx):
+    def _get_frame_filename_and_path(self, exception, frame_idx):
```

Comment:
Don't have to do it in this PR, but I find it very useful to try to add any files I touch to https://github.com/getsentry/sentry/blob/efa9c5e0bb2957b0c67bd18f551d502b15c4f38d/mypy.ini

so we can do type checking. More often then not, it catches something I miss.

Reply:
You could probably delete this function as well since it doesn't look like its used:
https://github.com/getsentry/sentry/blob/0df162af40176a170a3c2a9d41e2feb52a7b6c61/src/sentry/api/endpoints/source_map_debug.py#L259-L264

Reply:
yup! i just caught that as i got the notification for this comment , left in from an earlier version